--- moreutils/Makefile.orig	2013-11-06 03:35:50.000000000 +0100
+++ moreutils/Makefile	2014-03-03 07:37:30.628133713 +0100
@@ -5,7 +5,7 @@
 INSTALL_BIN?=install -s
 PREFIX?=/usr
 
-DOCBOOK2XMAN=xsltproc --param man.authors.section.enabled 0 /usr/share/xml/docbook/stylesheet/docbook-xsl/manpages/docbook.xsl
+DOCBOOK2XMAN=xsltproc --param man.authors.section.enabled 0 /usr/share/sgml/docbook/xsl-stylesheets/manpages/docbook.xsl
 
 all: $(BINS) $(MANS)
 
